pkg/cert/util.go

```diff
@@ -36,6 +38,9 @@ var ErrWrongPublicKey = core.NewError("failed to decode PEM block containing pub
 // ErrRsaPubKeyConversion indicates a public key could not be converted to an RSA public key
 var ErrRsaPubKeyConversion = core.NewError("Unable to convert public key to RSA public key", false)
 
+// ErrWrongPublicKey indicates a wrong certificate format
+var ErrWrongCertificate = core.NewError("failed to decode PEM block containing certificate", false)
```

Comment:
For me "wrong" implicates that it is a (valid) certificate, but the wrong one. I'd rename it to `ErrInvalidCertificate`